dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatCompletionServiceExtensions.cs

```diff
@@ -46,40 +45,38 @@ public static class ChatCompletionServiceExtensions
     }
 
     /// <summary>
-    /// Get a single chat message content for the prompt and settings.
+    /// Get a first chat message content for the prompt and settings.
     /// </summary>
     /// <param name="chatCompletionService">The target IChatCompletionSErvice interface to extend.</param>
     /// <param name="prompt">The standardized prompt input.</param>
     /// <param name="executionSettings">The AI execution settings (optional).</param>
     /// <param name="kernel">The <see cref="Kernel"/> containing services, plugins, and other state for use throughout the operation.</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
-    /// <returns>List of different chat results generated by the remote model</returns>
+    /// <returns>First chat message content for the prompt and settings.</returns>
     public static async Task<ChatMessageContent> GetChatMessageContentAsync(
         this IChatCompletionService chatCompletionService,
         string prompt,
         PromptExecutionSettings? executionSettings = null,
         Kernel? kernel = null,
         CancellationToken cancellationToken = default)
-        => (await chatCompletionService.GetChatMessageContentsAsync(prompt, executionSettings, kernel, cancellationToken).ConfigureAwait(false))
-            .Single();
+        => (await chatCompletionService.GetChatMessageContentsAsync(prompt, executionSettings, kernel, cancellationToken).ConfigureAwait(false))[0];
```

Comment:
Returning Single should not be an issue for the singular result extension, if you are getting an error you are misusing the `ResultsPerPrompt`.

This is a single to lead the caller to avoid non-intentional choice ignoring, where in this case we do recommend using the `plural` version `GetTextContents` like we have in `python`.